packages/commonwealth/server/controllers/server_communities_methods/update_community_id.ts

```diff
@@ -74,33 +74,21 @@ export async function __updateCommunityId(
     //  in the long-term. Alternative is to gradually duplicate the data
     //  and then delete the old data once redirect from old to new community
     //  is enabled
-    const models: ModelStatic<ModelInstance<{ community_id?: string }>>[] = [
-      // @ts-expect-error StrictNullChecks
+    const models: ModelStatic<
+      | ModelInstance<{ community_id: string }>
+      | ModelInstance<{ community_id?: string }>
```

Comment:
This should just be models where `community_id` is required not optional.

Reply:
I think you might be wrong about this.

update_community_id needs to change the old community_id to the new one and an object with an optional community_id would still have a stale one that would need to be updated.

Reply:
`community_id` is not optional in any of the models in the list. In fact, as I stated elsewhere, I have already fixed this type/issue in a separate PR.